app/models/submission.rb

```diff
@@ -94,7 +94,7 @@ def file_path
   end
 
   def filename_for_bulk_download
-		(users.map(&:tutorial_name).join('-') + '-' +
+		(team.first(180) + '-' +
```

Comment:
I'm not sure how you came up with 180.

Reply:
I roughly estimated that the ID and the Date and possibly 'LATE' should be around 60 characters. Maximum filename length in Windows seems to be 256. So I took 180 in case my estimate was rather bad :-) But feel free to make an exact count.